tests/python/unittest/test_numpy_op.py

```diff
@@ -829,6 +829,71 @@ def get_indices(axis_size):
                     assert_almost_equal(mx_out.asnumpy(), np_out, rtol=1e-3, atol=1e-5)
 
 
+@with_seed()
+@use_np
+def test_np_flip():
+    class TestFlip(HybridBlock):
+        def __init__(self, axis):
+            super(TestFlip, self).__init__()
+            # Necessary initialization
+            self.axis = axis
+
+        def hybrid_forward(self, F, x):
+            return F.np.flip(x, self.axis)
+
+    shapes1 = [(1, 2, 3), (1, 0), (3, 0, 2), (), (1,)]
+    shapes2 = [(2, 2, 3, 3), (1, 1, 2, 4)]
+    types = ['int32', 'int64', 'float16', 'float32', 'double']
+    for hybridize in [True, False]:
+        for oneType in types:
+            if oneType is 'float16':
+                rtol=1e-2
+                atol=1e-2
```

Comment:
why do you need higher tolerance for `float16` type? `flip` is a kind of copy, there should be no numerical instability issues.

Reply:
Fixed, thank you.